warehouse/query-core/src/main/java/datawave/query/function/JexlEvaluation.java

```diff
@@ -61,6 +63,12 @@ public JexlEvaluation(String query, JexlArithmetic arithmetic) {
 
         // Evaluate the JexlContext against the Script
         this.script = DatawaveJexlScript.create((ExpressionImpl) this.engine.createScript(this.query));
+
+        // Number of events evaluated
+        this.evaluatedCount = evaluatedCount;
+
+        // Number of events rejected
+        this.rejectedCount = rejectedCount;
```

Comment:
I think `documentsEvaluated` and `documentsRejected` are clearer variable names. Then you don't need a comment explaining what the variables are for.